package.json

```diff
@@ -10,15 +10,21 @@
     "install:extension-e2e": "cd apps/extension-e2e && yarn",
     "install:mobile": "cd apps/mobile && yarn",
     "install:libs": "cd libs/ui && yarn",
+    "install:backend": "yarn --cwd apps/backend",
+    "install:libs-be": "yarn --cwd libs/backend",
```

Comment:
rewrite other commands using `--cwd` flag please.  
and extend `install:all` command